### PR TITLE
Benchmarking scripts for MMS 1.0

### PR DESCRIPTION
**Pull request by piyushghai**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: mms/model_service_worker.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # pylint: disable=redefined-builtin
 
+import cProfile
```
Comment (vdantu) [inline mms/model_service_worker.py]: Why import this module in production? Have a method which can be called when "Benchmark" is set to True and import this from there.

File: ci/benchmark_ai.sh [hunk 2]
```diff
@@ -75,31 +72,19 @@ pull_docker_images(){
...
-        unzip -n JMeterPlugins-ExtrasLibs-1.4.0.zip -d $HOME/apache-jmeter-4.0 &&
-        unzip -n JMeterPlugins-WebDriver-1.4.0.zip -d $HOME/apache-jmeter-4.0 &&
-        cp ./lib/jmeter-plugins-cmn-jmeter-0.4.jar $HOME/apache-jmeter-4.0/lib/ &&
-        cp ./lib/ext/* $HOME/apache-jmeter-4.0/lib/ext
+        brew update || {
+        }
+        brew install jmeter --with-plugins || {
+        }
+
+        wget https://jmeter-plugins.org/get/ -O /$HOME/.linuxbrew/Cellar/jmeter/4.0/libexec/lib/ext/jmeter-plugins-manager-1.3.jar
```
[the remark this answers is not in the document]
Comment (zachgk) [reply ci/benchmark_ai.sh]: This is actually to ignore failure in the command with "set -e".  But, this file is not actually used anymore so I am deleting it.

File: mms/model_service_worker.py (hunk 1 above)
Comment (zachgk) [reply mms/model_service_worker.py]: Moved import to cProfile initialization

**Comment by vrakesh** [comment]
Look into the conflict and good to merge :)

**Pull request merged**